src/services/pagoPAProxyService.ts

```diff
@@ -93,7 +96,10 @@ export default class PagoPAProxyService {
               response.value.title || "Bad request (upstream)",
               response.value.detail || "Bad request response from upstream API"
             )
-          : unhandledResponseStatus(response.status)
+          : ResponsePaymentError(
```

Comment:
Why not use [this](https://github.com/pagopa/io-ts-commons/blob/79868ecb08a6d6b425f1522d8499eb2e86d28a0d/src/responses.ts#L262) or [this](https://github.com/pagopa/io-ts-commons/blob/79868ecb08a6d6b425f1522d8499eb2e86d28a0d/src/responses.ts#L508) ?

Reply:
Yes, I confirm.
Since the app shows different screens starting from the detail field we have to extend the error with an additional field `detail_v2` 
In that way the app versions that implement the handling of the new errors could use that new field.

If we use the same field `detail` to pass the new values, the app versions that do't implement the handling, will have a decoding error (In `PaymentProblemJson` `detail` and `detail_v2` is an enum)